libs/contrib/Data/Nat/Order/Properties.idr

```diff
@@ -105,3 +105,23 @@ multLteMonotoneLeft a b r a_lt_b =
   rewrite multCommutative a r in
   rewrite multCommutative b r in
   multLteMonotoneRight r a b a_lt_b
+
+-- If a number is both GTE and LTE to another number, then they are equal
+export
+oppositeLteEq : LTE a b -> GTE a b -> a = b
```

Comment:
This is the same as `antisymmetric` (https://github.com/idris-lang/Idris2/blob/main/libs/base/Data/Nat.idr#L114)